composeApp/src/androidMain/kotlin/com/wisemen/chargehub/ui/components/NextReservationComponent.kt

```diff
@@ -97,14 +100,18 @@ fun StartTimeInformation(startTime: String) {
 fun TimeSlotInformation(startHour: String, endHour: String) {
     Column {
         Text(
-            text = stringResource(R.string.jouw_tijdslot),
+            text = SharedRes.strings.your_timeslot.toStringResource(),
```

Comment:
is the stringResource() composable not working with this moko setup?

Reply:
No, moko uses its own tech to convert it to strings (as far as I was able to tell from tutorials), no tutorial used stringResource().